config.json

```diff
@@ -252,7 +252,15 @@
       "core": false,
       "difficulty": 4,
       "slug": "kindergarten-garden",
-      "topics": null,
+      "topics": [
+          "loops",
+          "arrays",
+          "strings",
+          "logic",
+          "pattern_recognition",
+          "enumerations",
+	  "lists"
```

Comment:
Thanks for adding the new topics @mhdsmr2! :) The indentation still isn't quite right, it should be indented two spaces after `topics`. Does that make sense? As soon as that's done then it's ready to merge! :)

Reply:
Ooh, that's a typo. Wait just a bit.

Reply:
Can you check again? It's looking like it's all good in my repo.